app/apollo/resolvers/cluster.js

```diff
@@ -236,9 +237,18 @@ const clusterResolvers = {
 
         checkComplexity( queryFields );
 
-        await validAuth(me, org_id, ACTIONS.READ, TYPES.CLUSTER, queryName, context);
+        let allAllowed = false;
+        try {
+          logger.info({req_id, user, org_id}, `${queryName} validating`);
```

Comment:
'validating' log message should be right after the `try {` line, so it 'includes' all the validation -- checkComplexity for example.